package-lock.json

```diff
@@ -19,20 +19,20 @@
       "integrity": "sha1-khpaE3RsIeMkRb8HmGgOnRGmUws=",
       "dev": true,
       "requires": {
-        "@babel/code-frame": "7.0.0",
-        "@babel/generator": "7.3.4",
-        "@babel/helpers": "7.3.1",
-        "@babel/parser": "7.3.4",
-        "@babel/template": "7.2.2",
-        "@babel/traverse": "7.3.4",
-        "@babel/types": "7.3.4",
-        "convert-source-map": "1.6.0",
-        "debug": "4.1.1",
-        "json5": "2.1.0",
-        "lodash": "4.17.11",
-        "resolve": "1.10.0",
-        "semver": "5.6.0",
-        "source-map": "0.5.7"
+        "@babel/code-frame": "^7.0.0",
```

Comment:
If I remember correctly, `^7.0.0` means any version newer than `7.0.0` (or equals to), and `npm install` will usually install the latest version (in a new machine that did not run `npm install` before), which may cause problems when the versions are not compatible, I have encountered this kind of problem before so I recommend to keep these unchanged, WDYT? @TinyAllen Did I miss anything?

Reply:
After executing the command 'npm i vue-js-modal', the 'package-lock.json' file is automatically modified. I don't know why it is automatically added with '^'.

Reply:
@TinyAllen any idea? Is this a matter? If not others look good to me

Reply:
@TinyAllen 
Or you tell me what parameters to add, npm can automatically add 'vue-js-moda' to the 'package-lock.json' file, without automatically adding the '^' symbol.

Reply:
@kezhenxu94 
Don't worry, I changed back.